packages/popover/src/Popover/Popover.spec.tsx

```diff
@@ -249,43 +270,57 @@ describe('packages/popover', () => {
     });
 
     describe(`when dismissMode=${DismissMode.Auto}`, () => {
-      // skip until JSDOM supports Popover API
-      // eslint-disable-next-line jest/no-disabled-tests
-      test.skip('dismisses popover when outside of popover is clicked', async () => {
+      test('dismisses popover when outside of popover is clicked', async () => {
+        const onToggleSpy = jest.fn();
         const { getByTestId } = renderTopLayerPopover({
           active: true,
           dismissMode: DismissMode.Auto,
+          onToggle: onToggleSpy,
         });
         const popover = getByTestId('popover-test-id');
 
         await waitFor(() => expect(popover).toBeVisible());
+        expect(onToggleSpy).toHaveBeenCalledTimes(1);
 
-        userEvent.click(document.body);
+        /**
+         * Simulate a toggle event to close the popover because even with the
+         * `@oddbird/popover-polyfill` there isn't a way to click outside of the
+         * popover to close it. This is currently tested in the Live Example story
+         */
+        fireEvent(
+          popover,
+          new ToggleEvent('toggle', {
+            bubbles: true,
+            oldState: 'open',
+            newState: 'closed',
+          }),
```

Comment:
is this worth testing then, since we're just testing that the toggle function is called when we trigger a toggle